### PR TITLE
Created landing page and moved page.tsx to /home

### PR DESCRIPTION
**Pull request by ZhenyuYu1**
Features added:

1. Dashboard Page.tsx moved to /home
2. Landing page created with a header, tutorial, and FAQ

### CODE REVIEW

File: src/app/create-event/test/page.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default function Test() {
```
Comment (pselvana) [inline src/app/create-event/test/page.tsx:1]: Is this file needed?

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,88 +1,31 @@
...
-      </div>{' '}
-      {/*div to close flex container*/}
-    </div>
+    <>
+      <Header />
+      <div className="mb-0 mt-10 pb-0">
+        <Intro />
+        <Image
+          src="/ViewingPage.png"
+          width="1000"
```
Comment (pselvana) [inline src/app/page.tsx:21]: Probably not the constraint you want because you are forcing this into a square and the aspect ratio is 1592x956 or 1.66.  I'm guessing auto might fix that.  The image itself is very large and a png makes it a slower download. Might be a good option for webm format instead or jpg with 80% or 95% compression.  This is landing page so you want responsive and fast and not have to wait for images to load.  I know this image will be replaced but want to get the concept right.

File: src/app/test/page.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+'use client'
```
Comment (pselvana) [inline src/app/test/page.tsx:1]: Needed?

File: src/components/Header.tsx [hunk 4]
```diff
@@ -1,44 +1,34 @@
...
-        <span className="h-6 rotate-45 border-l" />
-        <a href="https://nextjs.org/" target="_blank" rel="noreferrer">
-          <NextLogo />
-        </a>
+    <header className="w-full bg-base-200 p-4 shadow-md">
+      <div className="container mx-auto flex items-center justify-between">
+        <div className="flex text-xl font-bold">
+          <Image
+            src="/CalendarIcon.png"
+            width="1000"
```
Comment (pselvana) [inline src/components/Header.tsx:11]: this size is really odd here. It intrinsically will be much smaller and the actual file is 150px at best.  Needed?

File: src/components/Tutorial.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <div
+      id="Tutorial"
+      className="mb-5 mt-10 flex flex-col items-center justify-center"
+    >
+      <h1 className="mb-4 text-center text-4xl font-extrabold text-primary">
+        How it works:
+      </h1>
+      <ol className="list-inside list-decimal text-xl">
+        <li>
+          Create a new FindingATime event at &apos;Create Event&apos; using the
```
Comment (pselvana) [inline src/components/Tutorial.tsx:14]: There's a few of these in the code but do you mind changing this to "Finding A Time" from "FindingATime".
It's an attempt to make the phrase the entity and the action since appended together makes it look a bit clunky.

**Review by pselvana — CHANGES_REQUESTED** [review]
There's some create user changes in here so I'm not sure if that was because you re-used a branch and that's fine if that was the case.  I made a few asks for changes.  Great page framework and look!  I asked Alana and Jean-Luc to also provide some feedback.

File: src/app/create-event/test/page.tsx (hunk 1 above)
Comment (ZhenyuYu1) [reply src/app/create-event/test/page.tsx:1]: Removed the file I was using for testing.

File: src/app/test/page.tsx (hunk 3 above)
Comment (ZhenyuYu1) [reply src/app/test/page.tsx:1]: Removed the test page.

File: src/components/Tutorial.tsx (hunk 5 above)
Comment (ZhenyuYu1) [reply src/components/Tutorial.tsx:14]: I think the create user changes in here are because this branch is branched off the userLocalStorage branch. As for the name, should I change it to "Finding A Time" everywhere or leave it in the header as the app name as "FindingATime"?

Comment (pselvana) [reply src/components/Tutorial.tsx:14]: Grab the change in a follow up change you make. Let's get this in.

**Review by pselvana — APPROVED** [review]
Please follow up on any missing FindingATime concatenations in a future change. Also update that PNG later please.
